### PR TITLE
Add publish script and vsce to dev depends

### PR DESCRIPTION
**Pull request by calum-bird**

### CODE REVIEW

File: publish.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# Install dependencies and build our project
+./installDependencies.sh
+./buildAll.sh
+
+# Publish the extension. By default, VSCE will use the $VSCE_PAT
```
Comment (Coolroo) [inline publish.sh:7]: Is this something we need to configure?

Comment (calum-bird) [reply publish.sh:7]: Not sure what you mean by configure. The environment variable for publisher token?

Comment (calum-bird) [reply publish.sh:7]: To clarify: we will use this script in a GH Actions workflow where we can provide the publisher token securely. This cannot be leaked (would give access to Trelent's vsce publisher account) so we dont configure it code-side.

Comment (Coolroo) [reply publish.sh:7]: ah I see, alright LGTM then!

**Review by Coolroo — APPROVED** [review]

**Pull request merged**